hazelcast-spring/src/test/resources/com/hazelcast/spring/fullcacheconfig-applicationContext-hazelcast.xml

```diff
@@ -236,7 +237,11 @@
             </hz:map>
 
             <hz:cache name="testCache">
-                <hz:wan-replication-ref name="testWan" merge-policy="PUT_IF_ABSENT" republishing-enabled="false"/>
+                <hz:wan-replication-ref name="testWan" merge-policy="PUT_IF_ABSENT" republishing-enabled="false">
+                    <hz:filters>
+                        <hz:filter implementation="FILTER1"/>
```

Comment:
`implementation` is not available in hazelcast-config. Can't we make spring and hazelcast config sync in terms of xml configuration?